test/Status.test.ts

```diff
@@ -0,0 +1,123 @@
+import { BucketInfo, OriginalBucketInfo } from "../src/messages/BucketInfo";
```

Comment:
There's no need to create so many lines. Make the test concise. Check the status in Bucket and Entry tests. Don't create a new file just for this status – the change is too small.